templates/landing/homepage.html

```diff
@@ -17,7 +17,7 @@ <h1 class="font-weight-bold">Welcome To</h1>
         <div class="flex-column m-5">
           <h3 class="my-2">Choose University</h3>
           <div>
-            <select aria-label="choose-university" id="selectContent">
+            <select aria-label="choose-university" id="select_Content_university">
```

Comment:
It is the 5 time i am asking, 
Can you read universities from from DB? 
and through a view parse the data to template :) ?

Reply:
> It is the 5 time i am asking, Can you read universities from from DB? and through a view parse the data to template :) ?

well this is part of the problems we are having ..  :sweat_smile: 
once we understand how to do it correctly and make it work we will send a PR .. @Yarboa

Reply:
@Yarboa We can make the page read the data from the database but we don't know how to make it send us to the desird page..
For example -
Choosing a university from a drop down menu -> automaticaly get redirected to university page (aka 'university_name.html'

Reply:
Please refer this
https://spookylukey.github.io/django-views-the-right-way/context-data.html
Try to think how you query db calling the template and run for loop on date returned

Reply:
@razashtar93 any new commits on the road? 
Please add also tests with client fixture for the contact_us page

Reply:
@Yarboa please see what I wrote on slack

Reply:
> * now navbar present log in and register when user isn't logged in

@razashtar93 So please add it to the Description